queries.sql

```diff
@@ -8,3 +8,28 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT *FROM animals WHERE neutered = 'True';
 SELECT *FROM animals WHERE name != 'Gabumon';
 SELECT *FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+
+UPDATE animals SET species = 'unspecified';
+
+ROLLBACK;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon%';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+COMMIT;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth  > '2022-01-02';
+SAVEPOINT delete_2022;
+UPDATE animals SET  weight_kg = ( weight_kg * -1);
+ROLLBACK TO delete_2022;
+UPDATE animals SET  weight_kg = ( weight_kg * -1) WHERE id IN (7,5,8,6);
+COMMIT;
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts < 1;
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered, MAX(escape_attempts) FROM animals GROUP BY neutered;
+SELECT neutered, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY neutered;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=What%20is%20the%20average%20number%20of%20escape%20attempts%20per%20animal%20type%20of%20those%20born%20between%201990%20and%202000%3F), this query suppose to get the average number of escape attempts per `animal type` which means `species`, not `neutered`, So please fix that.
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=What%20is%20the%20minimum%20and%20maximum%20weight%20of%20each%20type%20of%20animal%3F), Please add a query that gets What are the minimum and maximum weight of each type of animal? Because you forgot to add it.